lsassy/writer.py

```diff
@@ -28,16 +31,27 @@ def get_output(self, out_format, users_only=False):
 
         return output_method.get_output()
 
-    def write(self, out_format="pretty", output_file=None, quiet=False, users_only=False, kerberos_dir=None):
+    def write(self, file_format, out_format="pretty", output_file=None, quiet=False, users_only=False, kerberos_dir=None):
         """
         Displays content to stdout and/or a file
         :param out_format: Output format
         :param output_file: Output file
+        :param file_format: File Logs Format
         :param quiet: If set, doesn't display on stdout
         :param users_only: If set, only returns users account, else returns users and computers accounts
         :return: Success status
         """
         output = self.get_output(out_format, users_only)
+        
+        if file_format is null:

```

Comment:
Why do you use `from sqlalchemy import null` for this, and not `None`?

Reply:
That's weird that I putted null instead of None.
Seems to be auto imported by my IDE. I'm gonna change that.